documentation/docs/examples/trusted-authorities.md

```diff
@@ -1,16 +1,27 @@
+---
+image: /img/integration-services/logo/integration_services.png
+description: The example-4 script authenticates an Integration Service client to manage Identities using the Admin identity created in example-0 and then creates and verifies identities and credentials.
+keywords:
+- create identity
+- create credential
+- validate credential
+- add trusted authority
+- example
+---
+
 # Trusted Authorities
 
 The [example-4](https://github.com/iotaledger/integration-services/blob/develop/clients/node/examples/4-TrustedAuthorities.ts)
-script authenticates a Integration Service client to manage Identities using the Admin identity created in [example-0](./how-to-run-examples) and then performs the following tasks:
+script authenticates an Integration Service client to manage Identities using the Admin identity created in [example-0](how-to-run-examples) and then performs the following tasks:
 
-* create a identity with username: `Driver`
-* create a credential for the `Driver` identity (root identity is a trusted authority)
-* verify credential issued by this trusted authority
-* check a credential (in the Tangle) that is issued by an unknown authority (it will be not valid)
-* add the external issuer as trusted authority
-* check same credential: now issuer is trusted and the credential is valid 
+1. Create an identity with username: `Driver`.
+2. Create a credential for the `Driver` identity (the root identity is a trusted authority).
+3. Verify credential issued by this trusted authority.
+4. Check a credential (in the Tangle) that is issued by an unknown authority (it will be not valid). 
+5. Add the external issuer as trusted authority.
+6. Check same credential, now issuer is trusted and the credential is valid.
```

Comment:
```suggestion
1. Creates an identity with username: `Driver`.
2. Creates a credential for the `Driver` identity (the root identity is a trusted authority).
3. Verifies a credential issued by this trusted authority.
4. Checks a credential (in the Tangle) that is issued by an unknown authority (it will be not valid). 
5. Adds the external issuer as trusted authority.
6. Checks same credential, now the issuer is trusted and the credential is valid.
```